governance/third-generation/aws/restrict-ec2-instance-type.sentinel

```diff
@@ -0,0 +1,32 @@
+# This policy uses the Sentinel tfplan/v2 import to require that
```

Comment:
This is SO much nicer than the older policy

Reply:
Thanks @scarolan